easybuild/tools/toolchain/__init__.py

```diff
@@ -50,6 +50,10 @@
 @author: Stijn De Weirdt (Ghent University)
 @author: Kenneth Hoste (Ghent University)
 """
+from pkgutil import extend_path
+
+# we're not the only ones in this namespace
+__path__ = extend_path(__path__, __name__)  #@ReservedAssignment
```

Comment:
@wpoely86, @JensTimmerman: putting this in place fixes the `import` problem that was caused by switching to `-m easybuild.main` rather than `easybuild/main.py`, but I'm not 100% sure why...

When I added a `print __file__` here, I'm seeing that this module (and `easybuild.tools`) is being imported as `easybuild/tools/toolchain/__init__.pyc`, while others (e.g. `easybuild.toolchain.cgmpich`) that are being imported print an absolute path.
There are other `__init__.py` files that do not have an `extend_path` though (e.g. `easybuild/framework/__init__.py`), they seem to be fine.

This may be related to the calls to the `import_available_modules` function made by the option parser, which is called for `easybuild.tools.repository`, `easybuild.tools.module_naming_scheme`, etc. to figure out the possible values for `--module-naming-scheme` for example.